backend/src/v5/services/filesManager.js

```diff
@@ -0,0 +1,81 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+const { getAllRemovableEntriesByType, getRefEntry } = require('../models/fileRefs');
+const FSHandler = require('../handler/fs');
```

Comment:
We just retrieve the methods we want to use

Reply:
Reason why It's not done it's because FSHandler and GridFSHandler have the same named functions (see ~L30)

so if I do that then i'd have to rename all the functions to avoid clash 

And as this file grow it'll very most likely be requiring all the functions from those files, so it's less of a concern